### DISCUSSION TITLE
what is your favorite programming language?

### DISCUSSION THREAD

**Discussion by tosinori-nt** [NAIST-SE/AttractingContributors#845] [2023 Attraction Contributions]
I am interested in your favorite programming languages!

Please send me a pull request with your favorite programming language in the "List of ProgrammingLanguage" in the Readme.md of the repository below!
https://github.com/tosinori-nt/AttractingContributors-ProgrammingLanguage/blob/main/README.md

**Comment by TakumaMiwa** [comment]
I made a pull request! Please help me too!
https://github.com/TakumaMiwa/AttractingContributors-Music
